root/etc/e-smith/events/actions/nethserver-nextcloud-conf

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
+source /opt/rh/rh-mariadb105/enable
+PATH=$PATH:/opt/rh/rh-mariadb105/root/usr/bin/
```

Comment:
I'm not sure, we just need to verify it.
Shouldn't be the PATH already prepared from the source command?
Is really needed the export below?

Reply:
done